frontend/src/components/plugins/tables/pluginActionsButtons.jsx

```diff
@@ -459,3 +472,100 @@ export function PlaybookFlowsButton({ playbook }) {
 PlaybookFlowsButton.propTypes = {
   playbook: PropTypes.object.isRequired,
 };
+
+export function MappingDataModel({ data, type, pythonModule }) {
+  // state
+  const [showModal, setShowModal] = React.useState(false);
+  const pythonModuleName = pythonModule.split(".")[0];
+
+  return (
+    <div className="d-flex flex-column align-items-center p-1">
+      <IconButton
+        id={`mapping-data-model__${pythonModuleName}`}
+        color="info"
+        size="sm"
+        Icon={VscJson}
+        title="View data model mapping"
+        onClick={() => setShowModal(!showModal)}
+        titlePlacement="top"
+        disabled={Object.keys(data).length === 0}
+      />
+      {showModal && (
+        <Modal
+          id="mapping-data-model-modal"
+          autoFocus
+          centered
+          zIndex="1050"
+          size="lg"
+          keyboard={false}
+          backdrop="static"
+          labelledBy="Data model modal"
+          isOpen={showModal}
+          style={{ minWidth: "50%" }}
+        >
+          <ModalHeader className="mx-2" toggle={() => setShowModal(false)}>
+            <small className="text-info">
+              Data model mapping
+              <MdInfoOutline
+                id="dataModelMapping_infoicon"
+                fontSize="16"
+                className="ms-2"
+              />
+              <UncontrolledTooltip
+                trigger="hover"
+                target="dataModelMapping_infoicon"
+                placement="right"
+                fade={false}
+                autohide={false}
+                innerClassName="p-2 text-start text-nowrap md-fit-content"
+              >
+                The main functionality of a `DataModel` is to model an
+                `Analyzer` result to a set of prearranged keys, allowing users
+                to easily search, evaluate and use the analyzer result.
+                <br />
+                For more info check the{" "}
+                <Link
+                  to="https://intelowlproject.github.io/docs/IntelOwl/usage/#datamodels"
+                  target="_blank"
+                >
+                  official doc.
+                </Link>
+              </UncontrolledTooltip>
+            </small>
+          </ModalHeader>
+          <ModalBody className="d-flex flex-column mx-2">
+            <small>
+              The <strong className="text-info">keys </strong>
+              represent the path used to retrieve the value in the analyzer
+              report and the <strong className="text-info">value</strong> the
+              path of the data model.
+            </small>
+            <small>
+              For more info check the{" "}
+              <Link
+                to={`https://github.com/intelowlproject/IntelOwl/blob/master/api_app/analyzers_manager/${type}_analyzers/${pythonModuleName}.py`}
```

Comment:
Can you check if it's possible in some way (I'm thinking about env var, but if you find another solution is still good) to customize the url ? In this way in case someone has a fork of the repo the link are not to our project but to the forked version ?